packages/server/src/__tests__/express4/expressSpecific.test.ts

```diff
@@ -193,7 +193,7 @@ it('supporting doubly-encoded variables example from migration guide', async ()
       query: 'query Hello($s: String!){hello(s: $s)}',
       variables: '{malformed JSON}',
     })
-    .expect(400, 'Unexpected token m in JSON at position 1');
+    .expect(400, /in JSON at position 1/);
```

Comment:
Error message changed in Node 20, this substring is compatible across all of our supported versions.